style.css

```diff
@@ -219,56 +238,63 @@ img {vertical-align: middle;}
 
 .about_us_A  {
 	margin-top: 50px;
-	padding: 35px 0px 35px 50px;
-	background-color: #BEB8D2;
+	background-color: white;
+	width: 100%;
 	grid-column-start: 2;
 	grid-column-end: 5;
 	text-align: center;
 	z-index: 30;
 	grid-row: 3;
-	height: auto;
+    float: left;
+	max-height: 300px;
+	max-width: 300px;
+	min-height: 300px;
+	min-width: 300px;	
 }
 
 .about_us_A img {
-	padding: ;
-	object-fit: cover;
-	height: auto;
-	margin: 0px 0px 0px 10%;
+    height: 100%;

```

Comment:
I made the image bigger, as i felt it would look better if it was bigger in the final outcome